tiatoolbox/cli/tile_server.py

```diff
@@ -0,0 +1,17 @@
+"""Command line interface for TileServer"""
+import click
+
+from tiatoolbox.cli.common import (
+    tiatoolbox_cli,
+)
+
+
+@tiatoolbox_cli.command()
+@click.argument("img_paths", nargs=-1, type=click.Path(exists=True))
```

Comment:
shouldn't this be using decorator from `cli/common.py` for image input? This will automatically add help for that function as well. https://github.com/TissueImageAnalytics/tiatoolbox/blob/446f2e1e05f79d4e397267a2dea4f962c506fdd1/tiatoolbox/cli/common.py#L30

Reply:
the img_input decorator will only allow a single path (to a file or directory). But the tileserver cli needs to be able to accept a variable number of paths.
for example if i had a slide and heatmaps from two different models i need to be able to do:

tiatoolbox tileserver path/to/slide path/to/heatmap1 path/to/heatmap2

to start the tileserver with the slide and two additional layers for the heatmaps

Reply:
You can still use `img_input` by defining defaults and options. See for example, https://github.com/TissueImageAnalytics/tiatoolbox/blob/c7fe7851cf78ab582e8e02bc1b21d5227063fc77/tiatoolbox/cli/common.py#L50 You can see different file types have been called here https://github.com/TissueImageAnalytics/tiatoolbox/blob/c7fe7851cf78ab582e8e02bc1b21d5227063fc77/tiatoolbox/cli/patch_predictor.py#L32 
Similarly, options for `mode` have been modified https://github.com/TissueImageAnalytics/tiatoolbox/blob/c7fe7851cf78ab582e8e02bc1b21d5227063fc77/tiatoolbox/cli/patch_predictor.py#L35

Reply:
the only options available for img_input is the help option. 

https://github.com/TissueImageAnalytics/tiatoolbox/blob/c7fe7851cf78ab582e8e02bc1b21d5227063fc77/tiatoolbox/cli/common.py#L30-L32

I could change that to allow more options to cover the useage needed here, if that would be a better way to do it?

Reply:
yes that's fine you can modify it by keeping the previous options as default options.

Reply:
Is this resolved?